langserver/signature.go

```diff
@@ -37,6 +37,28 @@ func (h *LangHandler) handleTextDocumentSignatureHelp(ctx context.Context, conn
 	if activeParameter > 0 {
 		activeParameter = activeParameter - 1
 	}
+
+	filename := h.FilePath(params.TextDocument.URI)
+	contents, err := h.readFile(ctx, params.TextDocument.URI)
+	if err != nil {
+		return nil, nil
+	}
+	offset, valid, _ := offsetForPosition(contents, params.Position)
+	if !valid {
+		return nil, nil
+	}
+	start := posForFileOffset(fset, filename, offset)
+	if start == token.NoPos {
+		return nil, nil
+	}
+
```

Comment:
All of the above is already done in `typecheck()`. 
We can just return the `start` value from `typecheck()`
But before I went about doing that I wanted to know if there is any simpler way of getting the `Pos` equivalent of given `lsp.Position` ?